.travis.yml

```diff
@@ -26,3 +26,6 @@ branches:
   - /^\d+\.\d+\.\d+(-SNAPSHOT|-alpha|-beta)?\d*$/ # trigger builds on tags which are semantically versioned to ship the SDK.
 after_success:
   - ./gradlew coveralls uploadArchives --console plain
+after_failure:
+  - cat /home/travis/build/optimizely/java-sdk/core-api/build/reports/findbugs/main.html
+
```

Comment:
nit. Remove extraneous line.

Reply:
this has been too useful to remove.  when travis fails findbugs for a JDK other than the one you are working on, this becomes invaluable in finding the reason for the failure.